### ISSUE TITLE
Azure Function .net 8.0 deploy error

### ISSUE THREAD

**Issue by eolamisan** [Azure/static-web-apps-cli#788]
**Are you accessing the CLI from the default port `:4280` ?**

- [ ] No, I am using a different port number (`--port`) and accessing the CLI from that port
- [X] Yes, I am accessing the CLI from port `:4280`

> Make sure you are accessing the URL printed in the console when running `swa start`!

**ℹ️ NOTE: Make sure to enable debug logs when running any `swa` commands using `--verbose=silly`**

**Describe the bug**
SWA cli does not allow to deploy an .net 8.0 Azure function.

**To Reproduce**

Provide "dotnet-isolated:8.0" as "apiRuntime" in file staticwebapp.config.json.

1. 
```
    "platform": {
      "apiRuntime": "dotnet-isolated:8.0"
    }

```
2. Type in command 'swa deploy --deployment-token xxx'
3. See error

**Expected behavior**
The API / Azure Function is deployed.

**Screenshots**

![image](https://github.com/Azure/static-web-apps-cli/assets/4191912/2809cc0e-ca71-410c-bad8-d302355ef763)

![image](https://github.com/Azure/static-web-apps-cli/assets/4191912/006c6ec4-3ead-4e46-a9dc-3206770d9216)

**Desktop (please complete the following information):**

- OS: Windows 11
- Version 23H2

**Additional context**

**Comment by jeroen-van-dijk** [comment]
I am facing the same issue while running `swa start` after upgrading to `apiRuntime` to `dotnet-isolated:8.0`

**Comment by cjk7989** [comment]
Hi @eolamisan and @jeroen-van-dijk, swa-cli@1.1.7 has been released which will support dotnet-isolated:8.0, thanks for reporting.
